models/report.js

```diff
@@ -26,9 +25,6 @@ var schema = new Schema({
   checkedOutAt: { type: Date, index: true }
 });
 
-// Give the report schema text search capabilities
-schema.plugin(listenTo);
```

Comment:
I investigated this comment thoroughly, and found no traces of how the `listen-to` custom plugin [written by an Aggie contributor](https://github.com/seelio/mongoose-listento) enables text search in the latest Mongoose.

The line below it, actually, is [_the_ way](https://docs.mongodb.com/manual/core/index-text/#overview) to support text search: you add a `text` index to a field in a MongoDB collection so that it is searchable via the [`$text`](https://docs.mongodb.com/manual/reference/operator/query/text/#op._S_text) query operator.